EIPS/eip-7212.md

```diff
@@ -108,17 +108,17 @@ The use of signature verification cost by `P256VERIFY` is `3450` gas. Following
 
 ## Rationale
 
-The "secp256r1" elliptic curve signatures consists of `v`, `r`, `s` components. Even if recovering the public key on the curve is possible, most of the applications are not generating `v` component of the signature and it causes an uncertainty of the result values. However, the signatures can be verified with only `r` - `s` values. In order to provide an exact and more compatible method, verification is preferred over recovery to propose in a precompiled.
+“secp256r1” ECDSA signatures consist of `v`, `r`, and `s` components. While the `v` value makes it possible to recover the public key of the signer, most signers do not generate the `v` component of the signature since `r` and `s` are sufficient for verification. In order to provide an exact and more compatible implementation, verification is preferred over recovery for the precompile.
 
-The signature values in `r` - `s` and the public key coordinates in the `x`- `y` provides direct computations in signing and verification part, so these formats are chose in the input data format which are 32 bytes.
+The signature values in `r` and `s` and the public key `(x, y)` provides direct computations in signing and verification part, so these formats are chosen in the input data format which are 32 bytes. (Nalin: ??? what does this mean ???)
```

Comment:
it's unclear to me what this paragraph means: would be nice to clarify

Reply:
i think they mean something like

> Existing P256 implementations verify (x, y, r, s) directly. We've chosen to match this style here, encoding each argument for the EVM as a `uint256`.

This is different from `ecrecover`, with the advantage that it 1. follows the NIST spec, 2. matches the rest of the (large) P256 ecosystem, and most importantly 3. allows execution clients to use existing well-vetted verifier implementations.

Reply:
Nice explanation. Thanks, added your paragraph as-is to this PR.